Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -312,6 +312,41 @@ class AppStoreConnectService {
         return output.map(Build.init)
     }
 
+    func removeBuildFromGroups(
+        bundleId: String,
+        version: String,
+        buildNumber: String,
+        groupNames: [String]
+    ) throws {
+        let appId = try ReadAppOperation(options: .init(identifier: .bundleId(bundleId)))
+            .execute(with: requestor)
+            .await()
+            .id
+
+        let buildId = try ReadBuildOperation(
+                options: .init(
+                    appId: appId,
+                    buildNumber: buildNumber,
+                    preReleaseVersion: version
+                )
+            )
+            .execute(with: requestor)
+            .await()
+            .build
+            .id
+
+        let groupIds = try groupNames.flatMap {
+            try ListBetaGroupsOperation(options: .init(appIds: [appId], names: [$0]))
+                .execute(with: requestor)
+                .await()
+                .map(\.1.id)
```

Comment:
If the tuple has named arguments you can use them unless there's some keypath limitation it should work
```suggestion
                .map(\.betaGroup.id)
```

If that doesn't work this should:
```suggestion
                .map { $0.betagroup.id }
```